pages/docs/end-user-guide/nominating.mdx

```diff
@@ -0,0 +1,129 @@
+
+- **Nominating**: Should you have already nominated and have that wallet open, will display this ico next to the validators you have nominated.
+
+- **Oversubscribed**: The "Oversubscribed" status appears when a validator accumulates over 256 active nominations. Although a validator may have numerous nominators shared with other validators, Avail consensus ensures an optimal distribution of nominators during elections to maximize AVL staking across the network. However, there are cases where a validator exceeds the 256 active nominations limit. In such instances, only the top 256 wallets will receive rewards, while wallets in the red section will not be eligible for any rewards.
+
+- **Slashed**: Validators can be slashed for various reasons, including downtime, misbehavior, or other protocol violations. Slashing involves a penalty, which can result in a loss of part or all all supporters (includig yours) stake.
+
+- **Block Nominations**: Block nominations indicates the validator is not looking for more nominations and has therefore blocked any community nominations.
+
+- **Era Points**: Era points are a measure of a validator's contribution to the network over a specific period (era). Validators earn era points based on their performance, including producing and validating blocks, and these points influence their overall reputation and rewards. 
+
+
+### Nominating VS Nomination Pools
+Nominating and nomination pools are both integral parts of the staking process in blockchain networks like Avail. However, they serve slightly different purposes and involve distinct mechanisms:
+
+- **Nominating**: Nominating refers to the process of selecting validators to delegate one's stake to in order to participate in the network's consensus mechanism and earn rewards. When users nominate validators, they essentially entrust them with their tokens to contribute to the security and operation of the network. By nominating validators that perform well and uphold the network's integrity, users can earn staking rewards in return for their contribution.
+
+- **Nomination Pools**: On the other hand, nomination pools are collective pools of staked tokens contributed by multiple users who have chosen to nominate a specific validator set. These pools aggregate the stakes of individual nominators and delegate them to the chosen validator. The primary purpose of nomination pools is to increase the nominator's chances of receiving rewards by combining their stakes with others, thereby enhancing the collective weight behind the Pools nomination. In essence, nomination pools allow smaller stakeholders to pool their resources together to compete with larger stakeholders and increase their chances of being selected to validate transactions and earn rewards.
```

Comment:
Do we have a number on how many nominators can be within a single pool?

Reply:
At this moment the chain is 200 per pool. I however have recommended the team to increase as it is extremely limiting. Other substrates that use this do not cap at 200